packages/usa-table/src/usa-table--stacked/usa-table--stacked.twig

```diff
@@ -15,25 +15,21 @@
         <th data-label="Document title" scope="row">Declaration of Independence</th>
         <td data-label="Description">Statement adopted by the Continental Congress declaring independence from the British Empire.</td>
         <td data-label="Year">1776</td>
-        <th data-label="Document Footer" scope="row">Footer</th>
```

Comment:
The stacked table variant does not accommodate the `tfoot` element, so I don't think we need to be concerned with testing the footer styles. 

Recommend we remove